scripts/stats/cam/exevs_cam_nam_firewxnest_grid2obs_stats.sh

```diff
@@ -39,26 +40,38 @@ do
      adate=`$NDATE -$fhr $datehr`
      aday=`echo $adate |cut -c1-8`
      acyc=`echo $adate |cut -c9-10`
+     if [ $acyc = 00 -o $acyc = 06 -o $acyc = 12 -o $acyc = 18 ]; then
      if [ -e $COMINnam/nam.${aday}/nam.t${acyc}z.${regionnest}.${outtyp}${fhr}.tm00.grib2 ]
      then
        echo $fhr >> $DATA/fcstlist
        let "fcstnum=fcstnum+1"
      else
-       if [ $acyc = 00 -o $acyc = 06 -o $acyc = 12 -o $acyc = 18 ]
-       then
+       echo $fhr >> $DATA/fcstmiss
+       let "fcstmiss=fcstmiss+1"
        if [ $SENDMAIL = "YES" ]; then
         export subject="NAM Firewx File Missing for EVS ${COMPONENT}"
         echo "Warning: The NAM Firewx file is missing for valid date ${VDATE}. METplus will not run." > mailmsg
         echo "Missing file is $COMINnam/nam.${aday}/nam.t${acyc}z.${regionnest}.${outtyp}${fhr}.tm00.grib2" >> mailmsg
         echo "Job ID: $jobid" >> mailmsg
         cat mailmsg | mail -s "$subject" $MAILTO
+       else
+        echo "WARNING: The NAM Firewx file is missing for valid date ${VDATE}. METplus will not run."
        fi
```

Comment:
Hi, @MarcelCaron-NOAA ,

Thank you for putting this line back into the code.  However, Alicia has stated that the full path needs to be in this WARNING statement.  Thus, it should read something like this:

"WARNING: File $COMINnam/nam.${aday}/nam.t${acyc}z.${regionnest}.${outtyp}${fhr}.tm00.grib2 is missing"  

Also, METplus will run if a single file is missing, but it won't run if many files are missing.  Thus, your warning message should not have "METplus will not run"  if a single file is missing.  It shouldn't run if all files are missing, though.  That is taken care of in a message below.  

@ShelleyMelchior-NOAA

Reply:
@MarcelCaron-NOAA I think you can leave that. My bigger concern is this is an "else" statement that depends on SENDMAIL. This statement should be written no matter what SENDMAIL is set to. Can we separate these WARNINGs from SENDMAIL entirely? @PerryShafran-NOAA @ShelleyMelchior-NOAA

Reply:
Yes please, remove the "METplus will not run" part of that line at the least.

Reply:
Full path added 👍

Reply:
@MarcelCaron-NOAA Yes, in that case you should just place that line after the let fcstmiss line and it would not be part of the SENDMAIL block.

Reply:
@MarcelCaron-NOAA Thanks for adding it the full path.  Now per Alicia's instructions, you need to move it out of the SENDMAIL block entirely.  I would place it after line 50, after the let fcstmiss statement.

Reply:
Excellent, thanks!